orebfuscator-common/src/main/java/net/imprex/orebfuscator/config/ProximityConfig.java

```diff
@@ -16,7 +16,7 @@ public interface ProximityConfig extends WorldConfig {
 
 	Matrix4f frustumCullingProjectionMatrix();
 
-	boolean useRayCastCheck();
+	boolean rayCastCheckEnabled();
```

Comment:
I still don't think this will compile since the `boolean rayCastCheckOnlyCheckCenter()` method is still missing in this interface.